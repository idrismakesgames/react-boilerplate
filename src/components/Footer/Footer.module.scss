.footer {
  background-color: #fff;
  margin-top: 200px;
  text-align: center;
}
