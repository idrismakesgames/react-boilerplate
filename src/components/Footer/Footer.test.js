import React from 'react';
import { shallow } from 'enzyme';
import Footer from './Footer';

describe('Footer', () => {
  const footer = shallow(<Footer />);

  it('renders properly', () => {
    expect(footer).toMatchSnapshot();
  });

  it('displays footer text', () => {
    expect(footer.find('.footer').text()).toEqual('Footer');
  });
});
