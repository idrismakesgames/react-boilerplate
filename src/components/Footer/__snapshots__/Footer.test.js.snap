// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders properly 1`] = `
<div
  className="footer"
>
  Footer
</div>
`;
